EIPS/eip-2462.md

```diff
@@ -0,0 +1,89 @@
+
+With this in mind, the Ethereum mainnet, a public chain, is used for popular purposes such as DID registry and verification. And Ethereum-compatible chains, which have the advantages of high TPS (!) And low gas cost (!), Can be used as side chains to quickly execute smart contract functions.
+
+And with Ethereum standard wallets, you can implement high-availability services that operate by delegating gas to service providers with minimal development costs.
+
+
+## Specification
+The example is based on using Klaytn's smart contract by signing with an Ethereum wallet.
+
+Here is an example of using an Ethereum wallet to execute a function in a smart contract in an Ethereum-compatible chain without gas costs.
+
+1. The JSON below is provided by QR code or as a function argument.
+
+```javascript
+{
+	chain: '',  // eth or klay
+	chainid: '', // 3(ropstn) or 1001(baobab)
+	address: '', // contract address
+	jsonInterface: {
+	 constant: false,
+	 inputs: [{
+	  name: 'uid',
+	  type: 'bytes32'
+	 }],
+	 name: 'checkin',
+	 outputs: [],
+	 payable: false,
+	 stateMutability: 'nonpayable',
+	 type: 'function'
+	},
+	inputs: [1]
+}
+```
+
+2. Sign in Ethereum.
+```javascript
+const timestamp = new Date().getTime()
+const fee = 0
+const data = web3.eth.abi.encodeFunctionCall(qr.jsonInterface, qr.inputs)
+const hex = data + web3.utils.padLeft(web3.utils.toHex(fee), 64).slice(2) + web3.utils.padLeft(web3.utils.toHex(timestamp), 64).slice(2)
+const pk = '[[privateKey]]'
+const signature = web3.eth.accounts.sign(web3.utils.keccak256(hex), pk).signature
+```
+
+3. Meta-tx
+
+When the signatures and parameters are sent to the Klaytn Service Provider's relayer, the relayer creates a transaction using its signature and parameters using the relayer's private key. Provider will use Klaytn's gas to run Klaytn's smart contracts.
+
+![eip](../assets/eip-2462/pic1.png)
+<p align="center"><img src="../assets/eip-2462/pic2.gif" alt="test" width="300"/></p>
+
+## Rationale
+In public chain Ethereum, gas costs can be reduced by using it for public purposes such as DID registration. Meta-tx can also be used to implement DID services that do not require any gas costs.
+
+Developers no longer need to develop separate wallets for each Ethereum-compatible chain, and users can also avoid the hassle of installing a dedicated wallet for each blockchain.
+
+Developers mix the pros and cons of Ethereum and Ethereum-compatible chain Can develop.
+
+Developers take advantage of the strengths and weaknesses of Ethereum and Ethereum-compatible chains, and services that are important and publicly needed, such as DID services, use the public chain Ethereum.
+
+And developers who want to provide gas-free services to their users with fast TPS and low-cost gas can quickly deliver usability by moving services to Ethereum-compatible chains without switching costs.
```

Comment:
This content is more appropriate for the Motivation section.  The rationale should indicate why specific choices within the specification were made, like why a particular structure instead of an alternative one, or why a particular variable name was chosen, or why a particular constant was chosen.  Motivation is where you discuss why someone would want to implement this specification.